drm-ffi/drm-sys/src/platforms/linux/x86_64/bindings.rs

```diff
@@ -1145,12 +1154,6 @@ pub struct drm_color_lut {
 }
 #[repr(C)]
 #[derive(Debug, Default, Copy, Clone, Hash, PartialEq, Eq)]
-pub struct hdr_metadata_infoframe__bindgen_ty_1 {
-    pub x: __u16,
-    pub y: __u16,
-}
-#[repr(C)]
-#[derive(Debug, Default, Copy, Clone, Hash, PartialEq, Eq)]
 pub struct drm_mode_crtc_page_flip {
```

Comment:
Not sure if this is an issue with our CI platform or bindgen, but why does this drop structs?

Reply:
This seems like a bug in the old bindgen, that struct is part of `struct hdr_metadata_infoframe` but without the rest of that struct it is completely useless.  Why the main struct isn’t part of the generated file, I don’t know, but nothing can regress from this upgrade.

Reply:
Ah, this probably happens, because we are [allow/block-listing types to generate bindings for](https://github.com/Smithay/drm-rs/pull/158/files#diff-87619c68d500c3b4ab86e2a26da6c6deb16df1f2d5222b7ebe03757919c3286cR102-R117) and that list doesn't contain the `hdr`-prefix. Would you mind adding that or opening an issue to track that?

Reply:
Done.